build.sbt

```diff
@@ -31,8 +66,8 @@ lazy val root = (project in file("."))
 lazy val example = project
   .dependsOn(pureconfig)
   .enablePlugins(MdocPlugin)
+  .settings(commonSettings)
```

Comment:
This change seems petty to me ;)

Reply:
yeah, I did it mindlessly when trying to concentrate :D